apps/server/src/modules/learnroom/controller/dto/room-element.url.params.ts

```diff
@@ -0,0 +1,20 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsMongoId } from 'class-validator';
+
+export class RoomElementUrlParams {
+	@IsMongoId()
+	@ApiProperty({
+		description: 'The id of the room.',
+		required: true,
+		nullable: false,
+	})
+	roomId!: string;
+
+	@IsMongoId()
+	@ApiProperty({
+		description: 'The id of the element within the room.',
+		required: true,
+		nullable: false,
+	})
+	elementId!: string;
+}
```

Comment:
Just one question: to avoid duplicate code, it would be better like this?

```suggestion
export class RoomElementUrlParams extends RoomUrlParams {
	@IsMongoId()
	@ApiProperty({
		description: 'The id of the element within the room.',
		required: true,
		nullable: false,
	})
	elementId!: string;
}
```

Reply:
Seperate elementId and roomId is more code but cleaner from readable and documentation side.